tsconfig.json

```diff
@@ -24,18 +24,21 @@
 	},
 	"include": [ "client", "server" ],
 	"exclude": [
-		"public",
-		"node_modules",
-		"**/node_modules/*",
-		"build",
-		"**/build/*",
 		"**/build-module/*",
 		"**/build-style/*",
+		"**/build/*",
 		"**/dist/*",
+		"**/node_modules/*",
+		"apps",
+		"build",
 		"cached-results.json",
-		"stats.json",
-		"server/bundlers/assets-*.json",
-		"server/devdocs/search-index.js"
+		"packages",
+		"public",
+		"server/bundler/assets*.json",
+		"server/devdocs/components-usage-stats.json",
+		"server/devdocs/proptypes-index.json",
+		"server/devdocs/search-index.js",
+		"stats.json"
```

Comment:
Should we perhaps just exclude `*.json`? Do we have any JSON files we want to check for types/autocomplete?

Reply:
I can imagine wanting to use package.json and schema json files with autocompletion and type information.

I'm inclined to live the selected JSON excludes if you don't feel strongly.

---

Aside: TypeScript supports JSON with `resolveJsonModule`, which doesn't seem to be included here… we should probably add it because we are allowing JSON imports in our build.

Reply:
Keeping things as-is sounds good if we have use-cases for building autocompletion from JSON files 👍